src/components/molecules/Wallet/Details.tsx

```diff
@@ -23,6 +27,28 @@ export default function Details(): ReactElement {
     setProviderInfo(providerInfo)
   }, [web3Provider])
 
+  async function addOceanToWallet() {
+    const wasAdded = await web3Provider.sendAsync({
+      method: 'metamask_watchAsset',
+      params: {
+        type: 'ERC20',
+        options: {
+          address: config.oceanTokenAddress,
+          symbol: networkName.localeCompare('Matic') === 1 ? 'mOCEAN' : 'OCEAN',
```

Comment:
Think it's safer to rely on the `networkId` here. Because we get this name from data in https://chainid.network, and I expect the Polygon people to rename their network over there in next weeks. If we base our condition on `networkId`, then we do not need to change anything when network names change. 

The `networkId` for Matic Mainnet is `137`